src/templates/attributor_list.json

```diff
@@ -90,22 +90,22 @@
             {
                 "name": "Marian Bienstein",
                 "institution": "Institute for Biotechnology, RWTH Aachen University",
-                "attribution": "Marian spontaneously gave us an introduction to the S1 lab in the Institute of Biotechnology and took his time to answer all our beginner’s questions and assist us with our first attempts at gel electrophoresis. Furthermore, he provided us with a PCR mastermix and loading dye at a time where we had not mastered the timing with our ordering process. We are grateful for his help in getting started in the lab."
+                "attribution": "Marian spontaneously gave us an introduction to the S1 lab in the Institute of Biotechnology and took his time to answer all our beginner's questions and assist us with our first attempts at gel electrophoresis. Furthermore, he provided us with a PCR mastermix and loading dye when we had not mastered the timing with our ordering process. We are grateful for his help in getting started in the lab."
             },
             {
                 "name": "Isabell Hofmann",
                 "institution": "Institute for Biotechnology, RWTH Aachen University",
-                "attribution": "Isabell provided us with our first TdT reaction primers before we were able to design primers ourselves. She always answered our questions and helped us find our way in the lab. We are grateful for her time and patience."
+                "attribution": "Isabell provided us with our first TdT reaction primers before we were able to design primers ourselves. She always answered our questions and helped us find our way into the lab. We are grateful for her time and patience."
```

Comment:
jetzt klingt es aber so, als hätten wir das Labor nicht gefunden und die hätte uns gezeigt, wo das ist xD Deshalb hab ich "in" geschrieben, weil ich damit meine, dass sie uns geholfen hat, uns im Labor zurechtzufinden. Du kannst das gerne irgendwie umschreiben